lib/embed/embed.dart

```diff
@@ -356,29 +363,44 @@ class PlaygroundMobile {
     Element rightPanel = $('#rightPanel');
     Element topPanel = $('#topPanel');
     Element bottomPanel = $('#bottomPanel');
-    Element toolbarLeftPanel = $('#toolbarLeftPanel');
-    Element toolbarRightPanel = $('#toolbarRightPanel');
     if (rightPanel != null && leftPanel != null && validFlex(h) != false) {
-      removeFlex(leftPanel);
-      removeFlex(rightPanel);
-      int l = roundFlex(double.parse(h));
-      leftPanel.classes.add('flex-${l}');
-      rightPanel.classes.add('flex-${10 - l}');
-      if (toolbarRightPanel != null && toolbarLeftPanel != null) {
-        removeFlex(toolbarLeftPanel);
-        removeFlex(toolbarRightPanel);
-        toolbarLeftPanel.classes.add('flex-${l}');
-        toolbarRightPanel.classes.add('flex-${10 - l}');
-      }
+      leftPanel.style.width = "$h%";
+      state['editor_split'] = new DSplitter($('vertical-splitter')).position;
```

Comment:
You'll probably want to use different names for the state settings names (`editor_split`, `output_split`). The desktop version of dartpad will share the same values with the embedded version; they probably don't want to share their splitter values.

Do you think it's important to persist the splitter positions across invocations of the embedded UI? It may not be necessary.